src/client/apps/edit/components/content/sections/related_articles/index.tsx

```diff
@@ -146,7 +160,17 @@ export class RelatedArticles extends Component {
           color={color}
           onChange={this.onAddArticle}
         />
-      </div>
+      </Flex>
     )
   }
 }
+
+const RelatedArticlePreview = styled.div`
+  div[class^="ArticleCard__ImageContainer"] {
```

Comment:
Is it always img content in this container? I think you're following BEM here for css naming conventions, in which case its not clear if ImageContainer correctly describes the Element

Reply:
Good point, would be smarter to export from reaction here and target `ImageContainer` -- I can do this as a follow up, for now it is able to target correctly.